qt/scientific_interfaces/test/ISISReflectometry/Batch/BatchPresenterTest.h

```diff
@@ -118,6 +118,46 @@ class BatchPresenterTest : public CxxTest::TestSuite {
     verifyAndClear();
   }
 
+  void testJobRunnerGetProcessAll() {
+    auto presenter = makePresenter();
+    // need to check m_jobRunner->m_processAll somehow
+    EXPECT_EQ(m_jobRunner->getProcessAll(), false);
```

Comment:
Could we use `TS_ASSERT_EQUALS` rather than `EXPECT_EQ` because it is what is used everywhere else?